src/components/views/elements/SettingsFlag.tsx

```diff
@@ -62,6 +62,17 @@ export default class SettingsFlag extends React.Component<IProps, IState> {
     }
 
     private getSettingValue(): boolean {
+        // If a level defined in props is overridden by a level at a high presedence, it gets disabled
+        // and we should show the overridding value.
+        if (
+            !!SettingsStore.settingIsOveriddenAtConfigLevel(
```

Comment:
don't need the `!!` here given its a boolean

Reply:
This is identified by Sonar: https://sonarcloud.io/project/issues?id=matrix-react-sdk&pullRequest=12593&resolved=false&sinceLeakPeriod=true